src/ert/config/ert_config.py

```diff
@@ -748,6 +759,9 @@ def handle_default(fm_step: ForwardModelStep, arg: str) -> str:
 
         job_list_errors = []
         job_list: List[ForwardModelStepJSON] = []
+        env_pr_fm_step = cls._uppercase_subkeys_and_stringify_subvalues(
+            ErtPluginManager().get_forward_model_configuration()
+        )
```

Comment:
Could we get this info in the `with_plugins` function instead of instantiating and fetching it directly here? Then we would get consistency where if you use the `with_plugins` version then plugins are loaded, while otherwise you just get a plain ert config.

Reply:
fixed.